locations/hoenn/encounters.json

```diff
@@ -0,0 +1,4824 @@
+          },
+          {
+            "name": "Cave Encounter 3, 25%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          },
+          {
+            "name": "Cave Encounter 4, 5%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          },
+          {
+            "name": "Surf Encounter 1, 100%",
+            "chest_unopened_img": "/images/locations/surfing.png",
+		        "chest_opened_img": "/images/locations/surfing_bw.png"
+          },
+          {
+            "name": "Old/Good Rod Encounter 1, 70/60%",
+            "chest_unopened_img": "/images/items/old_rod.png",
+		        "chest_opened_img": "/images/items/old_rod_bw.png",
+            "access_rules": ["old_rod","good_rod"]
+          },
+          {
+            "name": "Old/Good Rod Encounter 2, 30/20%",
+            "chest_unopened_img": "//images/items/old_rod.png",
+		        "chest_opened_img": "//images/items/old_rod_bw.png",
+            "access_rules": ["old_rod","good_rod"]
+          },
+          {
+            "name": "Good/Super Rod Encounter 3, 20/80%",
+            "chest_unopened_img": "/images/items/super_rod.png",
+		        "chest_opened_img": "/images/items/super_rod_bw.png",
+            "access_rules": ["good_rod","super_rod"]
+          },
+          {
+            "name": "Super Rod Encounter 4, 20%",
+            "chest_unopened_img": "/images/items/super_rod.png",
+		        "chest_opened_img": "/images/items/super_rod_bw.png",
+            "access_rules": ["super_rod"]
+          }
+        ],
+        "map_locations":
+        [
+          {
+            "map": "encountermap",
+            "x": 232,
+            "y": 84
+          }
+        ]
+      },
+      {
+        "name": "Desert Tunnel Encounters",
+        "access_rules": ["$defeat_champion,$fallarbor_access"],
+        "visibility_rules": ["$goal_not|champion"],
+        "sections":
+        [
+          {
+            "name": "Cave Encounter 1, 50%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          },
+          {
+            "name": "Cave Encounter 2, 34%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          },
+          {
+            "name": "Cave Encounter 3, 16%",
+            "chest_unopened_img": "/images/locations/cave.png",
+		        "chest_opened_img": "/images/locations/cave_bw.png"
+          }
+        ],
+        "map_locations":
+        [
+          {
+            "map": "encountermap",
+            "x": 28,
+            "y": 12
+          }
+        ]
+      },
+      {
+        "name": "Artisan Cave Encounters",
+        "access_rules": ["$battle_fronteir_access, wailmer_pail"],
```

Comment:
frontier spelled wrong. Also, similar question to the seafloor cavern as Bulbapedia only lists one encounter for the entire dungeon.

Reply:
Done. And yes, 1 encounter, Smeargle, in vanilla. But there are two floors, and each floor has a different single pokemon